src/plugins/ui_stateful/clipboard.ts

```diff
@@ -95,10 +96,12 @@ export class ClipboardPlugin extends UIPlugin {
           break;
         }
         const pasteOption = cmd.pasteOption || (this._isPaintingFormat ? "onlyFormat" : undefined);
-        this._isPaintingFormat = false;
         this.state.paste(cmd.target, { pasteOption, shouldPasteCF: true, selectTarget: true });
         this.lastPasteState = this.state;
-        this.status = "invisible";
+        if (!this.keepPaintingFormat) {
+          this._isPaintingFormat = false;
```

Comment:
```suggestion
          this.isPaintingFormat = false;
```
while you're touching this, remove the leading underscore, we don't use this for private attributes

Reply:
@hokolomopo There's a getter to get this attribute with the same name but without underscore.